src/vstest.console/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "vstest.console": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "VSTEST_RUNNER_DEBUG_ATTACHVS": "1",
```

Comment:
Actually this is on purpose, I want to use this in the end to attach to testhost and datacollector when you press debug on vstest.console in VS. But currently it does not work (because the vstest.console we build under VS is not complete and is missing the testhost runtime provider), so this just attaches to vstest.console if you press run without debugging :)
This is more like how it would look later. 
```
        "VSTEST_HOST_DEBUG_ATTACHVS": "1",
        "VSTEST_DATACOLLECTOR_DEBUG_ATTACHVS": "1",
```
I can remove this if you feel strongly about this :)